tox.ini

```diff
@@ -52,8 +52,10 @@ deps =
     pytest-django
     pytest-factoryboy
     pytest-mock
+    pytest-cov
 commands =
-    pytest {posargs}
+    pytest --cov=rgd --cov=core --cov=geodata {posargs}
+    coverage html
```

Comment:
I added this html report generation so I could easily see what's hitting/missing during development. If this is too much overhead for our testing script, I can let it be something I only do locally when developing

Reply:
Tools work for us.  If you find the html useful, then we should generate it.